scripts/all_configs.go

```diff
@@ -17,9 +17,14 @@ import (
 	"io"
 	"io/ioutil"
 	"net/http"
+	"os"
+	"path/filepath"
+	"runtime"
 
-	"github.com/evergreen-ci/evergreen/cli"
-	"github.com/jessevdk/go-flags"
+	"github.com/evergreen-ci/evergreen/operations"
```

Comment:
I wonder if we shuldn't just move this into the real binary? it basically depends on everything already.

Reply:
Yea, but it's little more involved since i'd need to write a new rest communicator method. Can you make a ticket, and assign it to me and I'll do it.